_snippets/graphql/create-thread.mdx

```diff
@@ -44,7 +51,12 @@ mutation createThread($input: CreateThreadInput!) {
         }
       }
     ],
-    "labelTypeIds": ["lt_01HB924PME9C0YWKW1N4AK3BZA"]
+    "labelTypeIds": ["lt_01HB924PME9C0YWKW1N4AK3BZA"],
+    "threadFields": [{
+      "key": "my_string_field",
+      "type": "ThreadFieldSchemaType.String",
```

Comment:
this looks a bit dodgy, is this tested wrt how you specify an enum when passing in variables?

Reply:
Woops you're right, copy and paste error